scripts/quests/ahtUrhgan/Saga_of_the_Skyserpent.lua

```diff
@@ -0,0 +1,141 @@
+-----------------------------------
+-- Saga of the Skyserpent
+-- Log: 6, Quest: 43
+-- Fari-Wari: !pos 80 -6 -137 50
+-- qm7:       !pos -11 8 -185 62
+-- Biyaada:   !pos -65.802 -6.999 69.273 48
+-----------------------------------
+require('scripts/globals/common')
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/titles')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.SAGA_OF_THE_SKYSERPENT)
+
+quest.reward =
+{
+    item = xi.items.IMPERIAL_GOLD_PIECE,
+    title = xi.title.SKYSERPENT_AGGRANDIZER
+}
+
+quest.sections =
+{
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Fari-Wari'] = quest:progressEvent(823, { text_table = 0 }),
+
+            onEventFinish =
+            {
+                [823] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 1)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Fari-Wari'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:event(829)
+                    elseif quest:getVar(player, 'Prog') == 2 and player:hasKeyItem(xi.ki.LILAC_RIBBON) then
```

Comment:
Especially with long conditionals, could you break out multiple conditions into a multiline style?